recipes-openxt/xenclient-cryptdisks/xenclient-cryptdisks/crypttab

```diff
@@ -1,4 +1,4 @@
 # <target name>	<source device>		<key file>                      <options>
 log             /dev/xenclient/log      /config/keys/log.key
 cores           /dev/xenclient/cores    /config/keys/cores.key
-swap            /dev/xenclient/swap     SWAP
+swap            /dev/xenclient/swap     SWAP                            --cipher aes-xts-plain64 --key-size 512 --hash sha512
```

Comment:
If, like myself, your first thought it "How is this possibly working?". [Here](https://github.com/OpenXT/xenclient-oe/blob/master/recipes-openxt/xenclient-cryptdisks/xenclient-cryptdisks/cryptdisks.initscript) is why.
This does not follow [crypttab(5)](https://www.freedesktop.org/software/systemd/man/crypttab.html).

While systemd has [systemd-cryptsetup-generator](https://www.freedesktop.org/software/systemd/man/systemd-cryptsetup-generator.html), for sysvinit I am only aware of [cryptdisk_start](https://packages.debian.org/sid/cryptsetup) which seems to be provided by Debian.

It would be nice to move away from this custom initscript if there is a replacement somewhere.

Reply:
Had a similar reaction to learning that we have a bespoke implementation that combines a "standard" name with similar-but-different file format.  This PR is compatible with the current implementation. Will file a ticket to look for a more standard alternative.